ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/committer/CommitContextSpec.scala

```diff
@@ -67,6 +67,31 @@ class CommitContextSpec extends WordSpec with Matchers {
     }
   }
 
+  "readAllFiltered" should {
+    "return keys matching the predicate and mark them as accessed" in {
+      val expectedKey1 = DamlStateKey.newBuilder.setContractId("a1").build
+      val expectedKey2 = DamlStateKey.newBuilder.setContractId("a2").build
+      val expected = Map(
+        expectedKey1 -> Some(aValue),
+        expectedKey2 -> None
+      )
+      val inputs = expected ++ Map(DamlStateKey.newBuilder.setContractId("b").build -> Some(aValue))
+      val context =
+        newInstance(inputs = inputs)
```

Comment:
Minor: naming the parameter does not improve readability here.

Reply:
You have to name it as it is the second optional parameter.